Cargo.toml

```
@@ -15,14 +15,14 @@ codegen-units = 1
 lto = true
 
 [build-dependencies]
-clap = { version = "3", features = ["cargo", "env"] }
+clap = { version = "3", features = ["cargo", "env", "derive"] }
 clap_complete = "3"
 
 [dependencies]
 anyhow = "1"
 bytes = "1"
 chrono = "0.4"
-clap = { version = "3", features = ["cargo", "env", "wrap_help", "deprecated"] }
+clap = { version = "3", features = ["cargo", "env", "wrap_help", "deprecated", "derive"] }
```

The `cargo` feature isn't needed anymore for the derive stuff. Also I sorted the keys from the build-dependencies and then appended the stuff that's not needed for the `build-dependencies` so the two dependency parts start the same and are more easily comparable for the human eye.

I am not sure about removing `deprecated` but I think its an independent question.

```suggestion
clap = { version = "3", features = ["derive", "env", "wrap_help", "deprecated"] }
```